jupyterlab/labbox_ephys_widgets_jp/src/extensions/curation/CurationSortingView/AllUnitsTab.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Implemented the AllUnits tab which shows the SortingView plugins. Maybe we want a better name than "All Units"